style.css

```diff
@@ -0,0 +1,245 @@
+@font-face {
+  font-family: 'Roboto';
+  src: url(../fonts/Roboto-Regular.ttf) format(ttf);
+}
+
+@font-face {
+  font-family: 'Crete Round';
+  src: url(fonts/CreteRound-Regular.ttf) format(ttf);
+}
+
+@font-face {
+  font-family: 'Poppins';
+  src: url(fonts/Poppins-Regular.ttf) format(ttf);
+}
+
+@font-face {
+  font-family: 'Inter';
+  src: url(fonts/Inter-VariableFont_slnt,wght.ttf) format(ttf);
+}
+
+@font-face {
+  font-family: 'InaiMathi';
+  src: url(fonts/Murasuit.ttf) format(ttf);
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please pay attention to the `font` and `space` indentation of your texts, because the font of the `h1` tag, `h2` tag, and `p` tag is not like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=xpoiaNr2EWdOHYE7-0) and it would be good if you give some padding between them 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/212731793-2fcf51be-305e-407b-b600-fe904908273c.png)

Reply:
- Your commitment to achieving a great design is really inspiring and I think you are almost there - just a little more tweaks 👌 
- [ ] Just to show you what your design looks like, I have attached another screenshots below 👇 which is just like the previous reviewer shared.

![image](https://user-images.githubusercontent.com/99715640/212759158-8bd9f870-d780-4b02-8495-925ec9b337e2.png)

- [ ] Your fonts are not really consistent with figms's design now but I see that you have imported the fonts. What I do notice though is that the paths are not really well set. Check again, and you fill see that your `font` folder is inside `assets` but the paths you imported from do not have the correct details.
- How about trying this path?
```
@font-face {
  font-family: 'Roboto';
  src: url(./assets/fonts/Roboto-Regular.ttf);
}

@font-face {
  font-family: 'Crete Round';
  src: url(./assets/fonts/CreteRound-Regular.ttf);
}

@font-face {
  font-family: 'Poppins';
  src: url(./assets/fonts/Poppins-Regular.ttf);
}

@font-face {
  font-family: 'Inter';
  src: url(./assets/fonts/Inter-VariableFont_slnt,wght.ttf);
}

/* Please check to be sure that you have this or that the template has this */
@font-face {
  font-family: 'InaiMathi';
  src: url(.assets/fonts/Murasuit.ttf); 
}
```
- Another solution to this problem would be to just go to [Google Fonts](fonts.google.com), search for the fonts, get the links and include them in your `style.css` or `index.html` file.
_Check [W3Schools](https://www.w3schools.com/css/css_font_google.asp) for more information on how to use Google fonts_